WordPress/src/main/java/org/wordpress/android/ui/posts/prepublishing/home/compose/PrepublishingHomeSocialItem.kt

```diff
@@ -90,10 +93,52 @@ private fun SocialContainer(
     }
 }
 
-@Preview
-@Preview(uiMode = UI_MODE_NIGHT_YES)
+private val lowOnSharesDescriptionStyle: TextStyle
+    @Composable
+    get() = MaterialTheme.typography.body2.copy(
+        color = AppColor.Yellow50,
+    )
+
+private val defaultDescriptionStyle: TextStyle
+    @Composable
+    get() = MaterialTheme.typography.body2.copy(
+        color = MaterialTheme.colors.onSurface.copy(alpha = ContentAlpha.medium),
+    )
+
+@Composable
+private fun DescriptionText(
+    text: String,
+    isLowOnShares: Boolean,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier,
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.spacedBy(Margin.Small.value)
+    ) {
+        if (isLowOnShares) {
+            Icon(
+                painterResource(R.drawable.ic_notice_white_24dp),
+                contentDescription = null,
+                tint = AppColor.Yellow50,
+                modifier = Modifier.size(24.dp),
```

Comment:
💡  Does it make sense to use `Margin.ExtraMediumLarge` instead of `24.dp` in this context?

Reply:
Since this is the icon size I believe we should use the actual size in this case, not the Margin constant.